keycard swarm.md

```diff
@@ -0,0 +1,102 @@
+---
+id: #000-template
+title: Swarm Proposal
+status: research/specification/implementation/maintenance
+lead-contributor
```

Comment:
same line, is YAML

same with contributors format
- foo
- bar

Reply:
corrected